### DISCUSSION TITLE
[General][Java SDK (Component)] Does dubbo's triple protocol have to use IDL mode to support http calls?？

### DISCUSSION THREAD

**Discussion by crazyStar00** [apache/dubbo#14007] [General]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

dubbo version: 3.3.0-beta.1 jdk:1.8 platform: windows 10

Using the dubco-samples-triple-no-idl project, it is found that after the service is started, it cannot be directly invoked through http. After the request is initiated, the server can receive the request from the client, but the client keeps pinding
However, a service registered via IDL using dubco-samples-triple-GRPC can accept and respond to http requests

Is it stated that triple protocol does not support pojo yet? I think it is to support 3.3.x, which version is probably supported?

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
Please try test on 3.3.0-beta.2

**Reply by crazyStar00** [reply]
3.3.0-beta.2 is ok.When will the next release of 3.3.0 be released?

**Reply by crazyStar00** [reply]
There might still be a problem
![image](https://github.com/apache/dubbo/assets/18380553/4e5e44a6-16bd-4b61-9cbd-99ce99bc03e0)
I understand that you should return “hello star“

**Reply by AlbumenJ** [reply]
We expect to release it in June.

**Reply by AlbumenJ** [reply]
Please submit a issue for this case.
